plugin-maven/src/main/java/com/diffplug/spotless/maven/FormatterFactory.java

```diff
@@ -56,6 +57,8 @@
 
 	private final List<FormatterStepFactory> stepFactories = new ArrayList<>();
 
+	private ToggleOffOn toggle;
```

Comment:
I've seen that you've used `@Nullable` in the Gradle plugin's `FormatExtension` but not here. May I suggest adding `@Nullable` here as well?

Reply:
Not sure why, but @Nullable blows up our maven build.  I've tried to add it, and it has caused problems.